### ISSUE TITLE
Encountered `exit` error when calling native executable 

### ISSUE THREAD

**Issue by fsat** [typesafehub/conductr-cli#280]
This error is encountered when attempting to exit with a non-zero exit code.

```
Felixs-MBP-2:Downloads felixsatyaputra$ ./sandbox stop
Error: Encountered unexpected error.
Error: Reason: NameError name 'exit' is not defined
Error: Further information of the error can be found in the error log file: /Users/felixsatyaputra/.conductr/errors.log
Failed to execute script sandbox

Felixs-MBP-2:Downloads felixsatyaputra$ ./conduct info
Error: Unable to contact ConductR.
Error: Reason: HTTPConnectionPool(host='127.0.0.1', port=9005): Max retries exceeded with url: /v2/bundles (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x108cff0b8>: Failed to establish a new connection: [Errno 61] Connection refused',))
Error: Start the ConductR sandbox with: sandbox run IMAGE_VERSION
Error: Encountered unexpected error.
Error: Reason: NameError name 'exit' is not defined
Error: Further information of the error can be found in the error log file: /Users/felixsatyaputra/.conductr/errors.log
Failed to execute script conduct
```

**Comment by fsat** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [ConductR CLI: Encountered `exit` error when calling native executable](https://trello.com/c/aSRMzSqQ/504-conductr-cli-encountered-exit-error-when-calling-native-executable)

**Comment by fsat** [comment]
Fixed by #286
